app/components/form/fields/DiskSizeField.tsx

```diff
@@ -57,6 +59,7 @@ export function DiskSizeField<
         // Run any additional validators passed in from the callsite
         return validate?.(diskSizeGiB)
       }}
+      disabled={disabled}
```

Comment:
it's already spread as part of `{...props}` if you delete this line and 37

Reply:
Actually, it looks like `TextFieldProps` already has `disabled?: boolean` on it (through a chain of inheritances I don't love). So you could get rid of all three lines in this file.

https://github.com/oxidecomputer/console/blob/644a45b8e4ab673ad51754e372277abc3ddfd036/app/components/form/fields/TextField.tsx#L30-L33

https://github.com/oxidecomputer/console/blob/644a45b8e4ab673ad51754e372277abc3ddfd036/app/components/form/fields/TextField.tsx#L24

https://github.com/oxidecomputer/console/blob/644a45b8e4ab673ad51754e372277abc3ddfd036/libs/ui/lib/text-input/TextInput.tsx#L35-L38

Amusingly it doesn't even need to be there, it seems, because it is already on the native `input` props. The code typechecks fine with it removed.

<img width="662" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/d69e3da2-10d5-4a45-af37-146cd8f2249e">

Reply:
good catch!